addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -502,7 +502,12 @@ msgctxt "#31107"
 msgid "WideList"
 msgstr ""
 
-#empty strings from id 31108 to 31109
+#empty strings from id 31108 to 31108
+
+#: /xml/VideoOSD.xml
```

Comment:
We use the fully qualified name here => addons/skin.estuary/xml...
I know this is not used correctly at other places in this file, but for new strings we should do it right,